src/index.test.js

```diff
@@ -32,7 +32,20 @@ describe('Tmate GitHub integration', () => {
     const customConnectionString = "foobar"
     execShellCommand.mockReturnValue(Promise.resolve(customConnectionString))
     await run()
-    expect(execShellCommand).toHaveBeenNthCalledWith(1, "pacman -Sy --noconfirm tmate")
+    expect(execShellCommand).toHaveBeenNthCalledWith(1, "pacman -Sy --noconfirm tmate");
+    expect(core.info).toHaveBeenNthCalledWith(1, `Web shell: ${customConnectionString}`);
```

Comment:
What I am concerned about a little bit is that main-loop test is duplicated many times. 
Main-loop implementation is not platform-specific, so it seems testing main-loop only once is sufficient.